### PR TITLE
test(e2e): Add additional assertions to aid end-to-end tests

### PR DESCRIPTION
**Pull request by teneightfive**
## Proposed changes

The current suite of end-to-end tests can sometimes fail unexpectedly.

This change attempts to wrap the logic around signing in and choosing
a location with more assertions to see where things are consistently
going wrong.

Code suggestion from @solidgoldpig

### CODE REVIEW

File: test/e2e/pages/page.js [hunk 1]
```diff
@@ -50,7 +50,29 @@ export default class Page {
   async chooseLocation() {
     const count = await this.nodes.locationsList.count
     const randomItem = Math.floor(Math.random() * count)
-    return t.click(this.nodes.locationsList.nth(randomItem))
+
+    await t.expect(count).notEql(0)
+    await t.expect(this.getCurrentUrl()).contains('/locations')
```
Comment (barbaracassani) [inline test/e2e/pages/page.js:55]: I am nitpicking here, but maybe check the url first and the dom rendering after? (e.g. switch the two statements).

Comment (solidgoldpig) [reply test/e2e/pages/page.js:55]: What @barbaracassani said

File: test/e2e/smoke.test.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ const usersWhoHaveADashboard = [
   },
 ]
 
-fixture('Smoke tests')
+fixture.only('Smoke tests')
```
Comment (barbaracassani) [inline test/e2e/smoke.test.js:49]: only?

File: test/e2e/pages/create-move.js [hunk 3]
```diff
@@ -291,7 +292,7 @@ class CreateMovePage extends Page {
     return fillInForm({
       dateFrom: {
         selector: this.fields.dateFrom,
-        value: faker.date.future().toLocaleDateString(),
+        value: format(faker.date.future(), 'd/M/yyyy'),
```
Comment (solidgoldpig) [inline test/e2e/pages/create-move.js:295]: @teneightfive @barbaracassani ^^

**Pull request merged**